openfl/interface/workspace.py

```diff
@@ -498,7 +503,12 @@ def open_pipe(command: str):
     echo('\n ✔️ DONE: Building base gramine-openfl image')
 
     workspace_path = Path.cwd()
-    workspace_name = workspace_path.name
+    if workspace_name is None:
+        workspace_name = workspace_path.name
+    else:
+        # Prevent potential path traversal issues:
+        # If workspace name is a path, use only the last element
```

Comment:
This is another surprising assumption 😁

Reply:
I agree with Igor, this might fail based on the `workspace_name` variable.